src/modules/dashboard/components/ExpenditurePage/Payments/Payments.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useCallback } from 'react';
+import classNames from 'classnames';
+
+import { defineMessages, FormattedMessage } from 'react-intl';
+import { FieldArray, useField } from 'formik';
+import Button from '~core/Button';
+import Recipient from '../Recipient';
+
+import styles from './Payments.css';
+import Icon from '~core/Icon';
+import { FormSection } from '~core/Fields';
+import { newRecipient } from './consts';
+
+const MSG = defineMessages({
+  payments: {
+    id: 'dashboard.Expenditures.Payments.defaultPayment',
+    defaultMessage: 'Add payments',
+  },
+  recipient: {
+    id: 'dashboard.Expenditures.Payments.defaultRrecipient',
+    defaultMessage: 'Recipient',
+  },
+  addRecipientLabel: {
+    id: 'dashboard.Expenditures.Payments.addRecipientLabel',
+    defaultMessage: 'Add recipient',
+  },
+});
+
+const Payments = () => {
+  const [, { value: recipients }, { setValue }] = useField('recipients');
+
+  const onToogleButtonClick = useCallback(
+    (index) => {
+      setValue(
+        recipients.map((recipient, idx) =>
+          index === idx
+            ? { ...recipient, isExpanded: !recipient.isExpanded }
+            : recipient,
+        ),
+      );
+    },
+    [recipients, setValue],
+  );
+
+  return (
+    <div className={styles.paymentContainer}>
+      <div className={styles.recipientContainer}>
+        <div className={styles.payments}>
+          <FormattedMessage {...MSG.payments} />
+        </div>
+        <FieldArray
+          name="recipients"
+          render={({ push, remove }) => (
+            <>
+              {recipients.map((recipient, index) => (
```

Comment:
We should be able to get the members just like we do with the `CreatePaymentDialog` component. Check the usage of `useMembersSubscription`. It may make this easier then using mock data here.